unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/BuildModeHUDView.cs

```diff
@@ -20,7 +19,9 @@ public class BuildModeHUDView : MonoBehaviour
     public Button tutorialBtn;
     public Button logOutBtn;
 
-    public GameObject publishGO, publishingGO, publishedGO;
+    public GameObject publishGO;
+    public GameObject publishingGO;
+    public GameObject publishedGO;
```

Comment:
What's the difference between those three? I think we should improve the naming here.